client/src/components/page/Sidebar/index.tsx

```diff
@@ -106,11 +108,27 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
               </Link>
             </Menu.Item>
           )}
-          <Menu.SubMenu key="admin-locations" title={LOCATIONS}>
-            <Menu.Item key="locations-unit">{LOCATIONS_UNIT}</Menu.Item>
-            <Menu.Item key="locations-unit-group">{LOCATIONS_UNIT_GROUP}</Menu.Item>
-            <Menu.Item key="locations-unit-group-set">{LOCATIONS_UNIT_GROUP_SET}</Menu.Item>
-          </Menu.SubMenu>
+          {ENABLE_PLANS && (
+            <Menu.Item key="plans">
+              <Link to={PLANS_LIST_VIEW_URL} className="admin-link">
+                {PLANS}
+              </Link>
+            </Menu.Item>
+          )}
+          {ENABLE_LOCATIONS && (
```

Comment:
We need to include this in the test suite since no test case accounts for the `ENABLE_LOCATIONS` is true. I did this while handling the [form configuration work](https://github.com/OpenSRP/web/blob/import-form-config/client/src/containers/ConnectedSidebar/tests/index.test.tsx#L51)

Reply:
addresses